.github/workflows/sycl-nightly.yml

```diff
@@ -50,6 +50,14 @@ jobs:
             target_devices: ext_oneapi_hip:gpu
             tests_selector: e2e
 
+          - name: Intel Build Tests Only
+            runner: '["Linux", "build"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:alldeps
+            image_options: -u 1001 --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:cpu
```

Comment:
does `target_devices` matter here or are we just giving it some dummy value?

Reply:
It is mostly a dummy value. However I think if there is none some AOT tests fail building (this is an area I have to do a lot more digging in tbh)